themes/dracula/dracula.css

```diff
@@ -0,0 +1,323 @@
+/*! =========================================== *
+ * DRACULA Theme
+ * Version: 3.14.0
+ * Author: Mike Barkmin
+ * Website: www.barkmin.eu
+ * Tweets @mikebarkmin
+ * =========================================== */
+.headerDesktop-container {
+  background-color: #282a36; }
+
+.search-form-fieldset {
+  background: #000; }
+
+[class*="searchForm-action-button-"] {
+  color: #D8D8D8; }
+
+.searchForm-action-button-advanced:hover, .searchForm-action-button-toggle:hover {
+  background: #FF79C6; }
+
+.customRadio-input:checked + .customRadio-mask {
+  box-shadow: inset 0 0 0 3px #fff, inset 0 0 0 10px #FF79C6;
+  border-color: #FF79C6; }
+
+.pm_button.primary, .pm_button.primary:active {
+  color: #D8D8D8;
+  background-color: #282a36;
+  border-color: #282a36; }
+
+.pm_button.primary:hover, .pm_button.primary:focus {
+  background-color: rgba(0, 0, 0, 0.7);
+  border-color: rgba(0, 0, 0, 0.7); }
+
+.navigation > li.active, .navigation > li:hover {
+  border-color: #FF79C6; }
+
+.navigation-link {
+  color: #D8D8D8; }
+
+.navigation-title:hover {
+  color: #FF79C6; }
+
+.pm_dropdown .navigationUser-logout {
+  background-color: #282a36 !important;
+  border-color: #282a36 !important;
+  color: #D8D8D8 !important; }
+
+.pm_dropdown .navigationUser-logout:hover {
+  background-color: #000 !important;
+  border-color: #000 !important;
+  color: #D8D8D8 !important; }
+
+.sidebar-btn-compose, .sidebar-btn-compose:active {
+  color: #D8D8D8 !important;
+  background: #FF79C6 !important;
+  border: none !important;
+  transition: background-color .2s ease-in-out; }
+
+.sidebar-btn-compose:hover {
+  color: #D8D8D8 !important;
+  background: #FF79C6 !important;
+  border: none !important;
+  box-shadow: inset 0 400px 400px rgba(0, 0, 0, 0.1); }
```

Comment:
I have noticed that the contrast ration is not the best for accessibility. Could you change this to another value please? An [`_override.scss` can be created to add theme specific styles](https://github.com/csalmeida/protonmail-themes/tree/master/templates/deutera_one) that do not follow the `_styles.scss` template.

The site recommends `rgb(216, 216, 216)` or `rgb(61, 61, 61)` if this helps but it is absolutely up to you!

<img width="1440" alt="Screenshot 2019-10-10 at 11 04 06" src="https://user-images.githubusercontent.com/12223613/66587391-fafabe00-eb92-11e9-89a9-7cb16a30463a.png">

> Color contrast test at https://accessible-colors.com

Reply:
Thanks for your suggestion. I have also checked other places where accessibility could be an issue and changes the color accordingly. I hope that everything is fine now.

Reply:
Awesome! LGTM 🚀